docs/source/alfacase_definitions/MultipleRunsDescription.txt

```diff
@@ -0,0 +1,18 @@
+.. rubric:: Definitions
+
+.. tab:: CaseDescription
+
+    .. parsed-literal::
+
+        class MultipleRunsDescription
+            variables: \ :class:`Dict <typing.Dict>`\[str, <class 'float'>] = {}
+            variations: \ :class:`Dict <typing.Dict>`\[str, dict[str, float]] = {}
+
+.. tab:: Schema
+
+    .. parsed-literal::
+
+            variables:  # optional
+                string: float
+            variations:  # optional
+                string: dict
```

Comment:
Do not use a "lazy" dict.


```suggestion
            variations:  # optional
                string:
                    string: number
```

Is the key of the first dict some user supplied "run identifier"?

Like:
```yaml
multiple_runs:
    variables:
        A: 1.1
        B: 2.2
    variations:
        small:
            A: 1.9
            B: 2.8
        big:
            A: 1000
            B: 2000
```